src/avram/upsert.cr

```diff
@@ -90,9 +90,26 @@ module Avram::Upsert
     end
   end
 
+  macro upsert_unique_on(*attribute_names)
```

Comment:
https://github.com/luckyframework/avram/pull/789/files#diff-a265a41187dff25dd916dbc390a4da9c8a879e991c7fad41300fcd7fc6200607R117 references the wrong method in the error